source/install/install-kubernetes-mattermost.rst

```diff
@@ -35,9 +35,26 @@ Save the file as ``mattermost-license-secret.yaml``. Apply the file, specifying
 
 **2. Create an Installation Manifest File**
 
-Open a text editor and create a text file with the following details.
+The Mattermost installation manifest contains fields which must be edited in line with your configuration and environment requirements.
+
+.. csv-table::
+    :header: "Field", "Description", "Must Edit"
 
-Save the file as ``mattermost-installation.yaml``:
+    "metadata.name", "The name of your Mattermost as it will be shown in Kubernetes. The shorter the better.", "Yes"
+    "spec.size", "The size of your installation. This can be '100users', '1000users, '5000users', '10000users', or '25000users'.", "Yes"
+    "spec.ingressName", "The DNS for your Mattermost installation.", "Yes"
+    "spec.version", "The Mattermost version.", "No"
+    "spec.mattermostLicenseSecret", "The name of the Kubernetes secret containing your license (e.g. mattermost-license-secret). Required for Enterprise deployments.", "Yes"
```

Comment:
I think I would technically consider the license to be something you don't have to edit considering you could just be running without a license.

```suggestion
    "spec.mattermostLicenseSecret", "The name of the Kubernetes secret containing your license (e.g. mattermost-license-secret). Required for Enterprise deployments.", "No"
```